src/clientiface.cpp

```diff
@@ -238,11 +239,9 @@ void RemoteClient::GetNextBlocks (
 
 			/*
 				Don't generate or send if not in sight
-				FIXME This only works if the client uses a small enough
-				FOV setting. The default of 72 degrees is fine.
+				NOTE This only works if client and server use the same
```

Comment:
Hmm... Seems like it would work fine as long as the server setting is greater of equal to the client's setting.

Tried with values up to 120 and it worked fine.